taskcluster/ci/ui-test-apk/kind.yml

```diff
@@ -351,3 +350,79 @@ tasks:
         worker:
             env:
                 GOOGLE_PROJECT: moz-fenix
+    fenix-arm-release:
+        attributes:
+            build-type: beta-firebase
+            shipping-phase: push
```

Comment:
@JohanLorenzo Just want to verify again if this is all that's needed to schedule this `task` to be part of a `Push` phase Graph (e.g, [here](https://firefox-ci-tc.services.mozilla.com/tasks/groups/SCnpCb59S0u2UaYbRkUZ4A)). 

Looking at `release-notify-push` `task` ([here](https://github.com/mozilla-mobile/firefox-android/blob/main/taskcluster/ci/release-notify-push/kind.yml)) for example, are there any other attributes needed?

For context, @sv-ohorvath looks for the `release-notify-push` email to signal QA to start testing and running automation on candidates (so this is automating some of that)

Reply:
Thanks for asking! You can test this out locally 😃 That's one of the perks of taskgraph.

1. First, create a python virtual environment.
2. Activate it
3. Install taskgraph: https://github.com/taskcluster/taskgraph/#installation
4. Run `taskgraph target-graph -p taskcluster/test/params/main-repo-promote-beta.yml` ➡️ make sure `ui-test-apk-fenix-arm-release` isn't there
5. Run `taskgraph target-graph -p taskcluster/test/params/main-repo-push-beta.yml` ➡️ this time, it should be here.
6. You can do the same with `main-repo-promote-release.yml` for the release graph (instead of the beta one)

You can even see the impact of your patch with `taskgraph target-graph -p taskcluster/test/params/main-repo-push-beta.yml --diff origin/main`, `origin` being the name of the remote repository that points to `mozilla-mobile/firefox-android`. You can figure it out with `git remote -v`